### ISSUE TITLE
The WMS Client test server does not support CORS

### ISSUE THREAD

**Issue by robinhoutmeyers** [opengeospatial/ets-wms-client13#66]
When you start a WMS Client test using https://cite.opengeospatial.org/teamengine, a test server is started that you need to contact: https://cite.opengeospatial.org/teamengine/monitor/ followed by a number. This server apparently does not support CORS, making it impossible to certify web applications that include a WMS Client. 

To verify this, you can use an online CORS tester (e.g. https://cors-test.codehappy.dev/) and fill in the GetCapabilities of the server. An example web application with a WMS client can be found here: https://dev.luciad.com/portal/packed-samples/ria/dataformats/index.html?webgl&reference=epsg:4978 (click on Connect to, select WMS and fill in the GetCapabilities URL of the server.

**Comment by dstenger** [comment]
Thank you for proposing this enhancement.
I will discuss it with my colleagues of the CITE team.

**Comment by dstenger** [comment]
Required code changes must be done in TEAM Engine code.
Pull request: https://github.com/opengeospatial/teamengine/pull/603

**Comment by dstenger** [comment]
@robinhoutmeyers The enhancement has been installed on Beta environment (https://cite.ogc.org/te2/). Can you please check if your problem is solved?

**Comment by robinhoutmeyers** [comment]
@dstenger I am indeed able now to use our web client to connect to the test server set up by the WMS Client test. However, I do have one strange observation: while the test was able to capture my GetCapabilities request, it didn't capture any of my GetMap requests. This is strange because the GetMap requests did work well: I could see the data in my client. Here is an example for the BasicPolygons data:

![image](https://github.com/opengeospatial/ets-wms-client13/assets/11718993/70a0c07f-dc72-45d3-8f51-9c4846dea662)

Here is one of the requests sent by our web client:

https://cite.ogc.org/te2/monitor/28?SERVICE=WMS&VERSION=1.3.0&LAYERS=cite:BasicPolygons&STYLES=default&FORMAT=image/png&TRANSPARENT=true&CRS=EPSG:3857&BBOX=-222638.98158654713,556591.8018932568,0,779230.7834798039&WIDTH=256&HEIGHT=256&REQUEST=GetMap

But the test continued to claim that no GetMap request was received. Perhaps this is due to the beta environment, not sure.

**Comment by dstenger** [comment]
@robinhoutmeyers
Thank you for the feedback.
I just tested the Beta test suite with QGIS 3.22 and the GetMap requests were received correctly.

Can you please do a re-test? If the problem continues to occur, I propose to open a new GitHub issue.

**Comment by robinhoutmeyers** [comment]
@dstenger I did a retest: the issue is still present, but I now found out that it is actually just a display issue, similar as https://github.com/opengeospatial/ets-wms-client13/issues/68. The console and the test summary (when the test is stopped) correctly show the expected results, so I indeed confirm the CORS issue fix works as expected (thanks!); however, the display that shows the test progress does not get updated properly during my tests. I'll add a comment to https://github.com/opengeospatial/ets-wms-client13/issues/68.
